units/check_blocked_tasks.go

```diff
@@ -126,6 +126,12 @@ func checkUnmarkedBlockingTasks(t *task.Task, dependencyCaches map[string]task.T
 
 	dependenciesMet, err := t.DependenciesMet(dependencyCaches)
 	if err != nil {
+		grip.Error(message.Fields{
```

Comment:
I would preference grip.Debug, since we're only putting this here to help us understand what's going on, and we already log this error separately